text/0055-nested-multi-sigs.md

```diff
@@ -1,35 +1,35 @@
-- Feature Name: nested_multi_sig
+- Feature Name: Nested Minting Multi-Signatures
 - Start Date: 2022-11-02
 - MCIP PR: [mobilecoinfoundation/mcips#0053](https://github.com/mobilecoinfoundation/mcips/pull/0053)
 - Tracking Issue: https://github.com/mobilecoinfoundation/mobilecoin/issues/2693
 
 # Summary
 [summary]: #summary
 
-We currently require multi-signatures for authorizing `MintConfigTx` and `MintTx` transactions. At the moment, each multi-signature is defined as requiring M-of-N signatures. The set of authorized signers (`SignerSet`) specifies the potential signers coupled with the threshold (the `M` in `M-of-N`). Each signer is identified by their Ed25519 pulic key.
+We currently require multi-signatures for authorizing `MintConfigTx` and `MintTx` transactions. At the moment, each multi-signature is defined as requiring M-of-N signatures. Each signer is identified by their Ed25519 public key.
 We would like to evolve this such that each member of a signer set can either be an individual Ed25519 public key, or another signer set.
 
 # Motivation
 [motivation]: #motivation
 
-Allowing each signer in a multi-sig `SignerSet` to either be an individual key or it's own M-of-N set allows us to better represent the real-world signature hierarchy that we would like to use for minting and minting configurations.
+Allowing each signer in a multi-signature to either be an individual key or it's own multi-signature allows us to better represent the real world signature hierarchy that we would like to use for minting and minting configurations.
 
 For example, when specifying the required signers for some arbitrary token, we might say that we want two entities to authorize a minting transaction:
 1. The MobileCoin foundation
 2. A liquidity provider that holds the backing asset for the token being issued
 
 Right now, there is no good way to specify this, unless each organization simply has a single key. That is suboptimal, since in reality the organization might prefer to have a few keys and require it's own M-of-N such that there is no single person in the organization that can authorize a mint. That is a more secure scheme.
 
-Following the change in this PR, it will become possible to specify such signer sets, where each signer can be either an individual key, or another signer set.
+Following the proposal in this MCIP, it will become possible to specify such multi-signatures, where each signer can be either an individual key, or another multi-signature.
```

Comment:
:upside_down_face: again reverting my original suggestion
```suggestion
Following the proposal in this MCIP, it will become possible to specify such signer sets, where each signer can be either an individual key, or another signer set.
```